aws-serverless-java-container-springboot3/src/main/java/com/amazonaws/serverless/proxy/spring/AwsSpringHttpProcessingUtils.java

```diff
@@ -180,4 +197,36 @@ private static <T> T readValue(String json, Class<T> clazz, ObjectMapper mapper)
 		}
 	}
 
+    static final String HEADER_KEY_VALUE_SEPARATOR = "=";
+    static final String HEADER_VALUE_SEPARATOR = ";";
+    static final String ENCODING_VALUE_KEY = "charset";
+    static protected Charset parseCharacterEncoding(String contentTypeHeader) {
```

Comment:
Similar code is in place here https://github.com/aws/serverless-java-container/blob/f85319ec59ad2b452bdef8db012d19d74510bbc3/aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpServletRequest.java#L372 
wouldn't it make more sense to centralize?

Reply:
The question for me is: where to centralize? ;) 

maybe a new class com.amazonaws.serverless.proxy.internal.HttpUtils? like SecurityUtils.
I tried looking at the code, but couldn't find anything to consider other than this method 

but it’s your project that you decide

Reply:
Yes it has to be a new class as we have two separate request flows (old request flow without Spring Cloud functions and new flow with Spring Cloud functions). I'm fine with the utils approach...